cmd/svcat/output/instance.go

```diff
@@ -17,12 +17,19 @@ limitations under the License.
 package output
 
 import (
+	"encoding/json"
 	"fmt"
 	"io"
 
+	"github.com/ghodss/yaml"
 	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
 )
 
+type instance struct {
+	Kind                    string `json:"kind"`
```

Comment:
Is this cruft from when you were trying to get kind working? I'm hoping it's not needed and we can circle back in a follow-on PR to figure out how to populate the type meta like kubectl does.

Reply:
Yep. Thought I had cleaned this one up!